includes/js/gameBoard.js

```diff
@@ -237,28 +237,28 @@ function gameBoard() {
 					var keyDownCode = this.keyQueue[key];
 
 					// shift combinations
-					if(keyDownCode == 539) {
+					if(keyDownCode === (500+config.key_right)) {
```

Comment:
As 539 keyCode doesn't exist, I assume 500 is "shiftModifier" which is just added to the keyCode when shift key is held down.

If you are already cleaning up and making things more readable, it would make sense to do either:
1. Have actual key events in the keyQueue so all the key press event information would be accessible here
2. Have "shiftModifier" as a config variable as well so the code would be understandable ie: `config.shiftModifier + config.keyRight`

Reply:
That makes perfectly sense and I will add option 2